apps/files_external/lib/Controller/AjaxController.php

```diff
@@ -56,27 +59,30 @@ private function generateSshKeys($keyLength) {
 	#[NoAdminRequired]
 	public function getSshKeys($keyLength = 1024) {
 		$key = $this->generateSshKeys($keyLength);
-		return new JSONResponse(
-			['data' => [
+		return new JSONResponse([
+			'data' => [
 				'private_key' => $key['privatekey'],
 				'public_key' => $key['publickey']
 			],
-				'status' => 'success'
-			]);
+			'status' => 'success'
```

Comment:
If reformatting then diff friendly?

```suggestion
			'status' => 'success',
```